styles.css

```diff
@@ -678,4 +678,549 @@ main hr {
 @media screen and (min-width: 768px) {
   /* Here starts the navbar */
 
-}
\ No newline at end of file
+  .navbar {
+    height: 72px;
+    position: fixed;
+    width: 100%;
+    z-index: 5;
+  }
+
+  .logo {
+    max-width: 200px;
+    height: 40px;
+    margin-left: 0;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    cursor: pointer;
+    color: #fff;
+  }
+
+  .logo-link {
+    flex-basis: 50%;
+    margin-left: 10%;
+  }
+
+  #desktop-menu {
+    display: flex;
```

Comment:
- [ ] You have used `display: flex` 44 times, which falls under the category of bad practising since you are repeating yourself a lot. I would suggest you create a class with a name similar to d-flex and use it whereever you need